crt_portal/cts_forms/templates/forms/report.html

```diff
@@ -92,25 +94,52 @@ <h3 class="margin-bottom-1">
               <em>{{ field.help_text }}</em>
               {{ field.errors }}
               <div class="margin-bottom-2">{{ field }}</div>
+              <em>{{ field.help_texts.below }}</em>
             </fieldset>
           {% endfor %}
         {% endfor %}
       {% else %}
+        {# This is for questions that don't need grouping #}
         {% for field in wizard.form.visible_fields %}
           <fieldset class="usa-fieldset">
-            <legend class="margin-bottom-0">{{ field.label_tag }}</legend>
-            {{ field.errors }}
+            <h3><legend class="margin-bottom-0 usa-prose-h3">{{ field.label_tag }}</legend></h3>
+            <div class="margin-bottom-2"><p class="usa-prose"><em>{{ field.help_text.0 }}</em></p>
+            {{ field.errors }}</div>
             <div class="margin-bottom-2">{{ field }}</div>
-            {{ field.help_text }}
+            {% if 'Details' == current_step_name %}
+               <span id="display_count">{{ field.help_text.1 }}</span><span id="count_message"> {{ field.help_text.2 }}</span>
+            {% endif %}
           </fieldset>
         {% endfor %}
       {% endif %}
-
+      <input type="submit" label="submit" value="{% trans "Next" %}" class="usa-button margin-bottom-1" />
       {% if wizard.steps.prev %}
-        <button name="wizard_goto_step" type="submit" label="first step" value="{{ wizard.steps.first }}" class="usa-button">{% trans "First" %}</button>
-        <button name="wizard_goto_step" type="submit" label="previous step" value="{{ wizard.steps.prev }}" class="usa-button">{% trans "Back" %}</button>
+        <br>
+        <button name="wizard_goto_step" type="submit" label="previous step" value="{{ wizard.steps.prev }}" class="look-like-link margin-bottom-9">{% trans "Back" %}</button>
       {% endif %}
-      <input type="submit" label="submit" value="{% trans "Next" %}" class="usa-button margin-bottom-9" />
+
     </form>
   </div>
+<!-- move this later -->
```

Comment:
Just thinking ahead here — I think we'll want to move this to the `/static` folder. 

https://docs.djangoproject.com/en/2.2/howto/static-files/

Reply:
Thank you!!